### PR TITLE
Dockerfile changes to add build_shared_lib

### PR DESCRIPTION
**Pull request by sfatimar**
2021_1 indentation changes to make sure that 2021_1 version builds. 
Dockerfile changes to include the flag build_shared_lib

**Motivation and Context**
- Why is this change required? What problem does it solve?
- If it fixes an open issue, please link to the issue here.

### CODE REVIEW

**Comment by MaajidKhan** [comment]
@sfatimar Looks good for me. we need to update the documentation as well.
1.
https://github.com/microsoft/onnxruntime/blob/master/BUILD.md#windows-6

2.
https://github.com/microsoft/onnxruntime/blob/master/BUILD.md#linux-6

3.
Also add the --build_shared_lib flag inside the charp docker file
https://github.com/microsoft/onnxruntime/blob/master/dockerfiles/Dockerfile.openvino-csharp

File: dockerfiles/Dockerfile.openvino [hunk 1]
```diff
@@ -84,7 +84,7 @@ RUN apt update && apt -y install --no-install-recommends apt-transport-https ca-
     git clone --recursive -b ${ONNXRUNTIME_BRANCH} ${ONNXRUNTIME_REPO} && \
     /bin/sh onnxruntime/dockerfiles/scripts/install_common_deps.sh && \
     cd onnxruntime/cmake/external/onnx && python3 setup.py install && \
-    cd ${MY_ROOT}/onnxruntime && ./build.sh --config Release --update --build --parallel --use_openvino ${DEVICE} --build_wheel && \
+    cd ${MY_ROOT}/onnxruntime && ./build.sh --config Release --update --build --parallel --use_openvino ${DEVICE} --build_shared_lib --build_wheel && \
```
Comment (MaajidKhan) [inline dockerfiles/Dockerfile.openvino:87]: Also add the same --build_shared_lib flag inside the csharp docker file as well.
https://github.com/microsoft/onnxruntime/blob/master/dockerfiles/Dockerfile.openvino-csharp

Comment (MaajidKhan) [inline dockerfiles/Dockerfile.openvino:87]: we need to update the documentation as well.
1.
https://github.com/microsoft/onnxruntime/blob/master/BUILD.md#windows-6

https://github.com/microsoft/onnxruntime/blob/master/BUILD.md#linux-6

**Comment by sfatimar** [comment]
Maajid please check again

File: dockerfiles/Dockerfile.openvino (hunk 1 above)
Comment (MaajidKhan) [reply dockerfiles/Dockerfile.openvino:87]: Looks good for me!!

**Comment by MaajidKhan** [comment]
Looks good for me sahar!!

**Comment by MaajidKhan** [comment]
This is a old PR. The changes from this PR were already merged to Msft repo long back in Feb.
so closing this PR.

**Pull request closed without merging**